src/lib/installer/crate_installer.rs

```diff
@@ -18,7 +18,8 @@ fn invoke_rustup_install(toolchain: &Option<String>, info: &InstallCrateInfo) ->
             let rustup_component_info = InstallRustupComponentInfo {
                 rustup_component_name: component.to_string(),
                 binary: Some(info.binary.clone()),
-                test_arg: Some(info.test_arg.clone()),
+                // InstallRustupComponentInfo only supports one argument right now.
```

Comment:
why not implement it as well?

Reply:
Right, so the reason I put it off was that InstallRustupComponentInfo has an additional requirement that it must also work when `test_arg` is None, in which case no installation tests are done and cargo-make will just always attempt to install.

I'll see if I can find a clean way to incorporate this logic.